session4-6/personalwebsite/dist/css/style.css

```diff
@@ -0,0 +1,99 @@
+* {
+    box-sizing: border-box;
+}
+.row::after {
+    content: "";
+    clear: both;
+    display: table;
+}
+[class*="col-"] {
+    float: left;
+    padding: 15px;
+}
+
+html,body {
+  margin:0px;
+  padding:0px;
+  font-family: cursive;
+  }
+ul {
+    background: #f5f5f5;
+   }   
+li {
+    list-style: none;
+    display: inline-block;
+    padding:10px;    
+}
+li a {
+    text-decoration: none;
+}
+li a:hover{
+	text-decoration: underline;
+}
+section {
+width:100%;
+clear:both;
+padding: 20px 0px;
+}
+article {padding: 20px;}
+a {
+    color: chocolate;
+}
+
+/* For mobile phones: */
+[class*="col-"] {
+    width: 100%;
+}
+
+.aside,section {
+    background-color: #33b5e5;
+    padding: 15px;
+    color: #ffffff;
+    text-align: center;
+    font-size: 14px;
+    box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24);
+}
+
+.header {
+    background-color: #9933cc;
+    color: #ffffff;
+    padding: 15px;
+}
+.footer {
+    background-color: #0099cc;
+    color: #ffffff;
+    text-align: center;
+    font-size: 12px;
+    padding: 15px;
+}
+
+@media only screen and (min-width: 600px) {
+    /* For tablets: */
+    .col-s-1 {width: 8.33%;}
```

Comment:
In sass we can use loop to create this. Preferable you should use bootstrap instead.